modules/unauthenticated-users-cluster-role-binding.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * authentication/impersonating-system-admin.adoc
+// * cicd/builds/triggering-builds-build-hooks.adoc
+// * authentication/tokens-scoping.adoc
+// * authentication/managing-oauth-access-tokens.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-cluster-role-bindings_{context}"]
+= Adding unauthenticated groups to `ClusterRoleBindings`
+
+As a cluster administrator, you can add unauthenticated users to the following ClusterRoleBindings in {product-title}.
+
+. `system:scope-impersonation`
+. `system:webhook`
+. `system:oauth-token-deleter`
+. `self-access-reviewer`
+
+[IMPORTANT]
+====
+Always verify compliance with your organization's security standards when modifying role bindings, particularly when including unauthenticated access.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* The OpenShift CLI (oc) is installed.
+
+.Procedure
+
+.Add unauthenticated groups to a `ClusterRoleBinding`
```

Comment:
Per the procedure template, there shouldn't be any other additional headings under the `.Procedure` heading

```suggestion

```